### ISSUE TITLE
ValuesAndExpressions::ProhibitVersionStrings should not warn on 'use 5.18.2;'

### ISSUE THREAD

**Issue by Tux** [Perl-Critic/Perl-Critic#655]
I am all in favor of forbidding version-strings, but
``` perl
use 5.18.2;
```
is too common (and useful) to forbid.
rewriting that to 'use 5.018002;' seems just stupid (in my perception).

can I have a configuration option to allow that just here?

**Comment by Tux** [comment]
ping?

**Comment by petdance** [comment]
Sorry, we're making a new drive to look at the issues queue.

**Comment by Tux** [comment]
May I draw your attention to this issue again?
I really don't want to use version strings, but whatever format I use in `use` this warning keeps bugging me.

**Comment by petdance** [comment]
I agree it would be a good fix.  It would be a big help if someone came up with a patch.  I don't have the free time right now to go into it.
